views/components/SignUpForm.js

```diff
@@ -1,39 +1,59 @@
 import React from 'react';
+import AlphaSignUpForm from './AlphaSignUpForm';
+import BetaSignUpForm from './BetaSignUpForm';
 import FormField from './FormField';
 
 const SignUpForm = props => {
-  const { header, subhead, partnerId } = props;
+  const { header, subhead, partnerId, hideAlpha, showBeta } = props;
```

Comment:
ooo i see. I would agree the less config needed the better. Then I wonder if that can be achieved with 1 less prop also, like with `if (showBeta) { view = betaView } else { view = alphaView }`? Otherwise what you have here makes sense and overall i like the approach :)